spec/javascripts/components/listing/eligibilitySection_spec.coffee

```diff
@@ -46,12 +46,22 @@ do ->
         ctrl = $componentController 'eligibilitySection', locals, {parent: fakeParent}
 
       describe 'occupancy', ->
-        it 'returns 1 for SRO', ->
-          unitSummary = { minOccupancy: 1 , maxOccupancy: 1 }
-          expect(ctrl.occupancy(unitSummary)).toEqual('1')
-        it 'returns a range for all other unit types', ->
-          unitSummary = { minOccupancy: 1 , maxOccupancy: 3 }
-          expect(ctrl.occupancy(unitSummary)).toEqual('1-3')
+        describe 'maxOccupancy is 1', ->
```

Comment:
nit, I think this one and is an SRO unit below are duplicates

Reply:
Done! see latest commit